titus_isolate/model/utils.py

```diff
@@ -82,8 +82,8 @@ def get_workload_from_disk(identifier) -> LegacyWorkload:
         workload_type = BURST
 
     opportunistic_cpus = 0
-    if WORKLOAD_JSON_OPPORTUNISTIC_CPU_KEY in passthrough_data:
-        opportunistic_cpus = passthrough_data[WORKLOAD_JSON_OPPORTUNISTIC_CPU_KEY]
+    if FENZO_WORKLOAD_JSON_OPPORTUNISTIC_CPU_KEY in passthrough_data:
```

Comment:
is this code path still being used for any Pods scheduled by the `kube-scheduler`? I see the function returns a `LegacyWorkload`, but I'm not familiar enough with the codebase to know which code paths will use it

Reply:
`LegacyWorkload` is now truly deprecated with the retirement of Mesos clusters.  All `Workloads` are now `KubernetesWorkloads`.